### PR TITLE
Fix/avoid test in prod config

### PR DESCRIPTION
**Pull request by pljarlegan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mediarithmics/plugins-nodejs-sdk",
-  "version": "0.7.10",
+  "version": "0.7.12",
```
Comment (nbelie) [inline package.json:3]: Why do you jump to v0.7.12 and not having the 0.7.11 ?

File: src/helpers/index.ts [hunk 2]
```diff
@@ -1 +1 @@
-export * from './ActivityAnalyzerTestHelper';
+// export * from './ActivityAnalyzerTestHelper';
```
Comment (nbelie) [inline src/helpers/index.ts:1]: If you do not export the helpers, how can you retrieve it outside ? 
As importing using the full is not the best idea IMO (cf. you change higher: 
`import * as helpers from "@mediarithmics/plugins-nodejs-sdk/lib/helpers/ActivityAnalyzerTestHelper";`
@PoulpiFr : any opinion ?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mediarithmics/plugins-nodejs-sdk",
-  "version": "0.7.10",
+  "version": "0.7.11",
```
Comment (PoulpiFr) [inline package.json]: https://docs.google.com/document/d/1BKV_oqmIFDwWZp3w-Ah3uz66keDYF0x0MNeHQn3lh1E/edit

it's not up to you to bump this

**Comment by dduponchel** [comment]
Checked with @nbelie, I'll close this pull request.

**Pull request closed without merging**